charts/dawarich/values.yaml

```diff
@@ -2,6 +2,22 @@ config:
   timezone: Europe/Amsterdam
   minimumMinutesSpentInCity: 60
   jobConcurrency: 10
+  oidc:
+    enabled: false
+    issuerUrl: https://example.com/
+    clientId: dawarich
+    clientSecret: your-client-secret
```

Comment:
I prefer not supporting plain text secret values at all, that also cuts down on helm templating